cmd/translate/main_grpc_integration_test.go

```diff
@@ -46,12 +46,23 @@ func randUploadRequest(t *testing.T, serviceID string) *translatev1.UploadTransl
 	return &translatev1.UploadTranslationFileRequest{
 		ServiceId: serviceID,
 		Language:  lang.String(),
-		Original:  gofakeit.Bool(),
 		Data:      data,
 		Schema:    schema,
 	}
 }
 
+func createUpdateTranslationReq(t *testing.T, serviceID string, override *translatev1.Translation,
```

Comment:
I think we prefix functions with `create` if it does send a GRPC request. And prefix with `rand` to generate a random entity.